Signal/src/ViewControllers/CallViewController.swift

```diff
@@ -345,42 +348,60 @@ class CallViewController: OWSViewController, CallObserver, CallServiceObserver,
 
 //        textMessageButton = createButton(imageName:"message-active-wide",
 //                                                action:#selector(didPressTextMessage))
-        audioSourceButton = createButton(imageName: "audio-call-speaker-inactive",
+        audioSourceButton = createButton(image: #imageLiteral(resourceName: "audio-call-speaker-inactive"),
```

Comment:
I'm not sure how I feel about these.

Reply:
It renders it in the IDE, which is cute, but not really that helpful. 

The main reason I like them is that they do a better job than logic like this:

```
 let image = UIImage(named: imageName)	
 assert(image != nil)
```

e.g. while developing you can be pretty sure you didn't mistype the name, and while running you know that it exists.